CONTRIBUTING.md

```diff
@@ -208,7 +208,7 @@ make sure you run all this from a filesystem that supports this.
    git submodule update
    ```
 
-5. Build your development environment with:
+5. Build your development environment with (make sure your user has access to Docker):
```

Comment:
That's mean, it's like saying: "Make sure you sacrificed the right :goat: " without telling you which :goat:...

And that you have to take care of this is mentioned in the linked openSUSE docu https://en.opensuse.org/SDB:Docker

Reply:
ok makes sense, i dropped it.

Reply:
You saved a lot of :goat: :goat: :goat: :smile: